### DISCUSSION TITLE
Difference between free with premium and pro?

### DISCUSSION THREAD

**Discussion by kud** [AppGround-io/bluetooth-keyboard-and-mouse-support#257] [Q&A] [answered]
Hey, I'd like to know the difference between the free version with the premium options paid and the pro version?

**Comment by AppGround-io** [comment, marked as answer]
The only difference is the color of the icon and the premium upgrade in the free version, otherwise they are the same.

**Reply by AdamKane41** [reply]
Lol

**Reply by kud** [reply]
> The only difference is the color of the icon and the premium upgrade in the free version, otherwise they are the same.

Okay thank you. :)

**Reply by Null-Fault** [reply]
Thank you. That's very generous of you.
